kernel/src/engine/arrow_utils.rs

```diff
@@ -662,6 +662,22 @@ fn parse_json_impl(json_strings: &StringArray, schema: ArrowSchemaRef) -> DeltaR
     Ok(concat_batches(&schema, output.iter())?)
 }
 
+/// write an arrow RecordBatch to a JSON string by appending to a buffer.
+///
+/// TODO (zach): this should stream data to the JSON writer and output an iterator.
+pub(crate) fn write_json(
```

Comment:
naming is hard, but maybe `to_json_string`? to me, "write" implies it's going to some output and not getting returned.

Reply:
oh yea sgtm thanks!

Reply:
or rather how about `to_json_bytes`?